README.md

```diff
@@ -34,6 +34,23 @@ Options to configure:
 
 Password should be a string `base64` encoded from `bcrypt` hash. You can use https://bcrypt-generator.com/ with default config and https://www.base64encode.org/.
 
+To include a custom template:
+- set the environment variable CADDY_SNIPPET to a file containining your custom
+  Caddyfile template.
+- pass your file into the container at the path specified by envvar CADDY_SNIPPET
+- it will be appended to the existing template when the container loads.
+
+Example:
+
+```yaml
+services:
+  caddy-gen:
+    volumes:
+      - ./caddy:/tmp/caddy
```

Comment:
Maybe we can use the full path to `/tmp/caddy/snippet` as a volume to match the example?

Reply:
I've found docker expects volumes to be directories, plus I've updated the example to mount two files. The key here is CADDY_TEMPLATE and CADDY_SNIPPET have to reference a file in the container's filesystem, however the user wants to get those files on there.